proto/plan_common.proto

```diff
@@ -190,3 +184,36 @@ message Cardinality {
 message ExprContext {
   string time_zone = 1;
 }
+
+message AdditionalColumnNormal {}
```

Comment:
What is `AdditionalColumnNormal`? ~~Please add some comments~~ Oops, do you mean "unspecified"? If so, I think we don't need such a `unspecified` option any more for `oneof`, please remove it.

Reply:
already removed